tests/schemas/test_property_schema.py

```diff
@@ -24,6 +24,21 @@ def test_valid_payload(self, create_property_manager):
         no_validation_errors = {}
         assert no_validation_errors == PropertySchema().validate(valid_payload)
 
+    def test_missing_required_parameters(self, create_property_manager):
+        invalid_payload = {
+            "created_at": Time.one_year_from_now(),
+            "updated_at": Time.today(),
```

Comment:
Let's get rid of `created_at` and `updated_at` here and also in the test above it (`test_valid_payload`).  These fields are created by the database and shouldn't be included in the payload.  Plus, using `Time.one_year_from_now()` isn't appropriate here (`created_at` shouldn't be one year after `updated_at` 🤔).  It must have slipped in with a previous PR.

Once those are removed, the `utils.time` import won't be necessary either, so let's get rid of that as well.  

Other than that, this test looks good 👍

Reply:
Ah yeah I just copied and pasted, whoops! I'll update those